acceptance/suites/tests/555-puppetserver-service/reboot.rb

```diff
@@ -0,0 +1,55 @@
+if (options[:type] == 'pe') then
+  pssn='pe-puppetserver'
+else
+  pssn='puppetserver'
+end
+
+test_name "Validates the ability of the #{pssn} service to reliably start after a reboot."
+
+max=3
+sleeptime=45
+retryCounter=0
```

Comment:
What's the expected impact on the overall runtime of our tests with this change?  I'm wondering what the best / expected / worst case situation is.  Best case seems like this test will take less than 60 seconds to complete (guessing), average looks to be about the same, but in the worst case I'm having a hard time understanding the impact on total test time.

Looks to be about 5 minutes, which is pushing the limits of what I'm comfortable with, but I'm not sure there's anything to be done.